sovtokenfees/doc/Interface/README.md

```diff
@@ -377,3 +377,71 @@ Example:
     }
 }
 ```
+
+## How to set fees for pool.
+ For setting fees into pool we need to making the next steps:
+ * Send a SET_FEES txn with appropriate amount for required alias. 
+ For example, we have an alias, like "add_new_steward" (and we want to set fees for adding new nym action). For setting fees for this alias, we need to send a SET_FEES transaction with map {"add_new_steward": 42}.
+ Txn will be looked as:
```

Comment:
Please mention that just setting fees without referencing them in AUTH_RULE doesn't have any effect.